src/uct/api/uct.h

```diff
@@ -712,7 +712,9 @@ enum {
     UCT_MD_FLAG_SOCKADDR   = UCS_BIT(7),  /**< MD support for client-server
                                                connection establishment via
                                                sockaddr */
-    UCT_MD_FLAG_INVALIDATE = UCS_BIT(8)   /**< MD supports memory invalidation */
+    UCT_MD_FLAG_INVALIDATE = UCS_BIT(8),  /**< MD supports memory invalidation */
+    UCT_MD_FLAG_WEAK_CACHE = UCS_BIT(9)   /**< MD registrations should not be
```

Comment:
send options to mailing list:
Option1: "negative" flag: UCT_MD_FLAG_WEAK_CACHE / UCT_MD_FLAG_DONT_CACHE
Option2: "positive" flag: UCT_MD_FLAG_LEAVE_PINNED / UCT_MD_FLAG_ALLOW_CACHE /...